mobile/library/common/http/client.cc

```diff
@@ -446,9 +447,9 @@ void Client::DirectStreamCallbacks::latchError() {
   }
 
   if (info.responseCodeDetails().has_value()) {
-    error_->message = Data::Utility::copyToBridgeData(info.responseCodeDetails().value());
+    error_->message = info.responseCodeDetails().value();
   } else {
-    error_->message = envoy_nodata;
+    error_->message = "";
```

Comment:
nit: entirely up to you, but this may read better as:
```
error_->message = info.responseCodeDetails().has_value() ? info.responseCodeDetails().value() : "";
```

Reply:
I changed it to this instead `error_->message = info.responseCodeDetails().value_or("")`